packages/yoastseo/src/languageProcessing/languages/es/helpers/internal/getParticiples.js

```diff
@@ -12,7 +12,8 @@ function isParticiple( word ) {
 	const participleSuffixes = [ "a", "o", "as", "os" ];
 	// For each participle suffixes, check if the word ends in one of them.
 	return participleSuffixes.some( suffix => {
-		if ( word.endsWith( suffix ) ) {
+		// Make sure only possible participles are targeted (minimum length being "> 3")
+		if ( word.length > 3 && word.endsWith( suffix ) ) {
 			// If the word ends with one of the suffixes, retrieve the stem.
 			const stem = word.slice( 0, -( suffix.length ) );
 			// Check if the stem is in the list of participles: return true if it is, otherwise return false.
```

Comment:
Not related to code written in this PR. But I was thinking that the phrasing _the list of participles_ is slightly inaccurate. What do you think of replacing it with _the list of verb stems_ or _participle stems_?

Reply:
Also applies to the other getParticiples files.

Reply:
I agreed. To make it more clear, you can update it indeed to something like: _the list of participle stems_.